pkg/devfile/adapters/common/types.go

```diff
@@ -27,6 +27,23 @@ type Storage struct {
 	Volume DevfileVolume
 }
 
+// BuildParameters is a struct containing the parameters to be used when building the image for a devfile component
+type BuildParameters struct {
+	Path            string                  // Path refers to the parent folder containing the source code to push up to a component
+	DockerfileBytes []byte                  // DockerfileBytes is the contents of the project Dockerfile in bytes
+	EnvSpecificInfo envinfo.EnvSpecificInfo // EnvSpecificInfo contains infomation of env.yaml file
+	Tag             string                  // Tag refers to the image tag of the image being built
+	IgnoredFiles    []string                // IgnoredFiles is the list of files to not push up to a component
+}
+
+// DeployParameters is a struct containing the parameters to be used when building the image for a devfile component
```

Comment:
`BuildParameters` and `DeployParameters` comment line reflect the same comment. Do you mean `// DeployParameters is a struct containing the parameters to be used when deploying the image for a devfile component` here?